tests/network/conftest.py

```diff
@@ -295,19 +297,23 @@ def _verify_jumbo_frame():
     def _verify_sriov():
         if any(test.get_closest_marker("sriov") for test in collected_tests):
             LOGGER.info("Verifying if the cluster supports running SRIOV tests...")
-            if not Namespace(name=py_config["sriov_namespace"], client=admin_client).exists:
+            if not sriov_namespace.exists:
                 failure_msgs.append(
-                    f"SRIOV operator is not installed, the '{py_config['sriov_namespace']}' namespace does not exist"
+                    f"SRIOV operator is not installed, the '{sriov_namespace.name}' namespace does not exist"
```

Comment:
If this condition is false, then there's no need to check the following conditions. Append the failure message and return.